Google.Api.Gax.Grpc/Rest/RestChannel.cs

```diff
@@ -75,7 +75,7 @@ public RestChannel(RestServiceCollection serviceCollection, string endpoint, Cha
                 }
                 cancellationTokenSource = new CancellationTokenSource(delayInterval);
             }
-            var httpResponseTask = SendAsync(restMethod, host, options, request, cancellationTokenSource.Token);
+            var httpResponseTask = ReadResponseAsync(SendAsync(restMethod, host, options, request, cancellationTokenSource.Token, HttpCompletionOption.ResponseContentRead));
```

Comment:
I think it would be clearer to introduce a new local variable for the sending task.

Reply:
👍